test-data/unit/check-enum.test

```diff
@@ -1971,8 +1972,8 @@ class B2(A2):  # E: Cannot extend enum with existing members: "A2"
 class A3(Enum):
     x: Final[int]  # type: ignore
 class B3(A3):
-    x = 1  # E: Cannot override final attribute "x" (previously declared in base class "A3")
-
+    x = 1  # E: Cannot override final attribute "x" (previously declared in base class "A3") \
```

Comment:
Note that this (and other equivalent) changes is correct: as per the [spec](https://typing.python.org/en/latest/spec/enums.html#defining-members), unannotated attrs should be interpreted as nonmembers, so, strictly speaking, you're assigning a soon-to-become enum instance to something of type `int`, breaking the interface. And it doesn't matter much as there is already another diagnostic on this line.